README.md

```diff
@@ -299,21 +300,21 @@ vdk --help</code></pre>
 
 <br/><br/>
 <p align="center">
-    <a href="#intro"><img src="https://img.shields.io/badge/VDK%20SDK-lightblue?style=plastic" alt="Intro to VDK SDK"></a>
-    <a href="#ingestion"><img src="https://img.shields.io/badge/Data%20Ingestion-lightblue?style=plastic" alt="Ingestion"></a>
-    <a href="#transformation"><img src="https://img.shields.io/badge/Data%20Transformation-lightblue?style=plastic" alt="Transformation"></a>
-    <a href="#deployment"><img src="https://img.shields.io/badge/Data%20Job%20Deployment-lightblue?style=plastic" alt="Job Deployment"></a>
-    <a href="#operations"><img src="https://img.shields.io/badge/Data%20Job%20Operations-lightblue?style=plastic" alt="Job Operations"></a>
-    <a href="#plugins"><img src="https://img.shields.io/badge/Lego%20like%20Extensibility-lightblue?style=plastic" alt="Extensibility"></a>
-    <a href="#contributing"><img src="https://img.shields.io/badge/Contributing%20&%20Contacts-blue?style=plastic" alt="Contributing & Contracts"></a>
+    <a href="#intro"><img src="https://img.shields.io/badge/VDK%20SDK-lightblue" alt="Intro to VDK SDK"></a>
+    <a href="#ingestion"><img src="https://img.shields.io/badge/Data%20Ingestion-lightblue" alt="Ingestion"></a>
+    <a href="#transformation"><img src="https://img.shields.io/badge/Data%20Transformation-lightblue" alt="Transformation"></a>
+    <a href="#deployment"><img src="https://img.shields.io/badge/Data%20Job%20Deployment-lightblue" alt="Job Deployment"></a>
+    <a href="#operations"><img src="https://img.shields.io/badge/Data%20Job%20Operations-lightblue" alt="Job Operations"></a>
+    <a href="#plugins"><img src="https://img.shields.io/badge/Lego%20like%20Extensibility-lightblue" alt="Extensibility"></a>
+    <a href="#contributing"><img src="https://img.shields.io/badge/Contributing%20&%20Contacts-blue" alt="Contributing & Contracts"></a>
```

Comment:
Because you renamed the heading this anchor link no longer works. 

YOu need to change "#contributing" to "#support-and-contributing" here and everywhere